Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -41,6 +41,10 @@ public struct Platform: Encodable {
     /// The Android platform
     @available(_PackageDescription, introduced: 5.2)
     public static let android: Platform = Platform(name: "android")
+
+    /// The WASI platform
+    @available(_PackageDescription, introduced: 5.2)
```

Comment:
Can you make this 999.0 as master is no longer tracking Swift 5.2. We'll replace all APIs with 999.0 when the version of Swift is announced.

Reply:
Should we call the property `webAssembly` to be more discoverable?

(The string bit needs to stay as "wasi", though.)

Reply:
that would be fine with me but maybe we should look at some other build system/package manager that supports web assembly and see what they're using as the public facing name?